debug.h

```diff
@@ -28,16 +28,15 @@ void end();
         if (!(exp)) {                                              \
             printf("Assert Failed (%s:%d): ", __FILE__, __LINE__); \
             printf(__VA_ARGS__);                                   \
-            printf("\n\n\nlooping...\n");                          \
-            end();                                                 \
+            exit(-1);                                               \
         }                                                          \
     }
 
 #define FATAL(...)                                      \
     {                                                   \
         printf("Error(%s:%d): \n", __FILE__, __LINE__); \
         printf(__VA_ARGS__);                            \
-        end();                                          \
+        exit(-1);                                        \
```

Comment:
From what I recall, there was a reason for the inf loop here. How does the board react when you `exit(-1)` @tolauwae ?

Reply:
`abort()` is called and the board reboots. If it's more desirable to have the board caught in an inf loop instead, I can just conditionally compile with end() if `ARDUINO` is defined and else with `exit(-1)`.

Reply:
If added the conditional compilation in c5a3be4.
@beardhatcode is this ok?